### ISSUE TITLE
Import Not Working?

### ISSUE THREAD

**Issue by Larnen** [Mordonus/RaidOps#335]
In v2.27e

Uploaded tonights data to the website. I then had a hard WS crash so the data in the addon was wiped back to the previous relog. For the life of me I cannot get the data re-imported from the website, something I have done many times. It just reloads at the end and seems unchanged.

Process is:

1) Download data from website
2) Hit Export/Import
3) Select 'Import Database'
4) Have tried having 'Website Export' on and off just to test in case that changed import format too - no difference
5) Paste file into buffer and hit the +
6) Check number of characters imported. Delete this number from the downloaded file
7) Repeat step 6 until fully imported (its 3 pastes for me)
8) Hit Export/Import

It then reloads and nothing has changed.

**Comment by Drutol** [comment]
I've just imported your data without an issue , if string is invalid it'll just reload and do nothing. Maybe multiple copy errors? ^^

**Comment by Larnen** [comment]
Wish there was another way to import, but I guess it can't read files on disk?

I've done the same as I normally do but still get issues. One oddity is that normally the 1st import is 31748 characters, and then next one is the same. This time for whatever reason the 2nd import fills the buffer to 63492, 4 characters short of what I would have expected.

One way to check the import is to check the EP value for "Bogdan Z" its 1400 on the website but still 0 in the addon. If youve managed to get it to import at 1400 in game, there must be some subtlety that Im missing, but as mentioned i've imported without error multiple times, 1st issue was in 2.27e.

Will try a re-download in case the download was corrupted for some reason

**Comment by Larnen** [comment]
Re-downloaded and same result. Is there anything group related that needs to be done differently?

**Comment by Drutol** [comment]
Nothing has been changed near import method...

**Comment by Drutol** [comment]
I've loaded your xml file , downloaded data from website , imported and Bogdan z has 1400 EP 1500 GP.

**Comment by Drutol** [comment]
90465 characters = length of import string

**Comment by Drutol** [comment]
I'm importing this differently though. Instead of pasting whole file I'm just selecting chunk of less than 30k characters , pasting in-game , hit + button , select another chunk of < 30 characters and add this to already added chunk.

**Comment by Drutol** [comment]
If you are looking for great text editor and don't want to use notepad++ , here's one http://www.sublimetext.com/3

**Comment by Larnen** [comment]
Maybe thats the difference. I normally paste the whole string in and hit +. The input buffer truncates to 31748. I then delete the 1st 31748 from the file, select all and + again. It then hit 63492. So I undelete and then redelete 63492, select all, and hit + a final time.

Then with Import Database selected, I hit Export/Import. This has always worked until now, although as mentioned its normally the same buffer size each time (31748) but for some reason the 2nd import only does 31744 (to 63492 total).

**Comment by Larnen** [comment]
Finally got it imported. Two possible issues, which one isn't clear:

1) At Mordonus's suggestion I imported 30k instead of max buffer size
2) Also noticed that when importing there was a newline being added. This time I deleted that and what seems to be an invisible character before it. Ie hit delete for the newline and then delete again and no change of cursor position

Doing these two worked. Issue 2 may be something specific to either where the breaks fell  in this specific xml or the way vim treats end of file.

Both weren't issues previously so not sure why there has needed a change of behaviour now, but the data was imported - thanks Mordonus :)
